ocw_data_parser/ocw_data_parser.py

```diff
@@ -458,9 +470,13 @@ def upload_course_image(self):
                     d = base64.b64decode(image_binary_data)
                     s3_bucket.put_object(Key=self.s3_target_folder + filename, Body=d, ACL="public-read")
                     log.info("Uploaded %s", filename)
-                    self.course_image_s3_link = bucket_base_url + filename
-                    self.course_image_alt_text = safe_get(file, "description")
-                    self.master_json["image_src"] = self.course_image_s3_link
-                    self.master_json["image_description"] = self.course_image_alt_text
-                    log.info("Uploaded %s", filename)
-                break
+                    if uid == self.course_image_uid:
+                        self.course_image_s3_link = bucket_base_url + filename
+                        self.course_image_alt_text = safe_get(file, "description")
+                        self.master_json["image_src"] = self.course_image_s3_link
+                        self.master_json["image_description"] = self.course_image_alt_text
+                    else:
+                        self.course_thumbnail_image_s3_link = bucket_base_url + filename
+                        self.course_thumbnail_image_alt_text = safe_get(file, "description")
```

Comment:
This seems to be the same data as the course image description, which is pulled directly from 0.json's `image_description` property.  I'm wondering if that's a more reliable source for the data, rather than the description field on each individual file.  Maybe we should ask Peter what OCW's process is for this, and have a fallback to use the course image description by default?

Reply:
I saved `thumbnail_image_description` in the master_json to match `image_description`.  I don't think this field is currently used by the ocw-to-hugo parser. `image_caption_text` and `image_alternate_text` which are extracted from the course json are used instead